tornjak-frontend/src/redux/actions/types.ts

```diff
@@ -140,10 +143,14 @@ export interface SpireHealthCheckingAction extends Action<typeof GLOBAL_SPIRE_HE
     payload: boolean;
 }
 
-export interface SpireHealthCheckTimeAction extends Action<typeof GLOBAL_SPIRE_HEALTH_CHECK_Time> {
+export interface SpireHealthCheckTimeAction extends Action<typeof GLOBAL_SPIRE_HEALTH_CHECK_TIME> {
     payload: SpireHealtCheckFreq;
```

Comment:
nit, doesn't have to be this PR, but maybe we should fix this typo wherever it is relevant?

Reply:
resolved